.github/workflows/ci.yml

```diff
@@ -20,6 +20,15 @@ env:
   GOSUMDB: off
 
 jobs:
+  PrTitleLint:
+    name: pr title lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
```

Comment:
fetch-depth might be the default value?

Reply:
The full-depth may be too large.

Reply:
@xuthus5 
**Only a single commit is fetched by default, for the ref/SHA that triggered the workflow.**

So I think we can omit this config

Reply:
OK,I found a project to verify whether context.payload requires git history. Maybe this step is redundant.